### PR TITLE
MINOR: Fix indentation for several doc pages

### PR DESCRIPTION
**Pull request by jlprat**
Fixes the indentation of the code listings for:
* api.html
* configuration.html
* design.html
* implementation.html
* toc.html

These changes consist of whitespaces added or removed for consistency. It also contains a couple of fixes on unbalanced html tags.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/toc.html [hunk 1]
```diff
@@ -86,26 +86,26 @@
...
-			<li><a href="#multitenancy-georeplication">Multi-Tenancy and Geo-Replication</a></li>
-			<li><a href="#multitenancy-more">Further considerations</a></li>
+                        <li><a href="#multitenancy-overview">Multi-Tenancy Overview</a></li>
+                        <li><a href="#multitenancy-topic-naming">Creating User Spaces (Namespaces)</a></li>
+                        <li><a href="#multitenancy-topic-configs">Configuring Topics</a></li>
+                        <li><a href="#multitenancy-security">Securing Clusters and Topics</a></li>
+                        <li><a href="#multitenancy-isolation">Isolating Tenants</a></li>
+                        <li><a href="#multitenancy-monitoring">Monitoring and Metering</a></li>
+                        <li><a href="#multitenancy-georeplication">Multi-Tenancy and Geo-Replication</a></li>
+                        <li><a href="#multitenancy-more">Further considerations</a></li>
```
Comment (jlprat) [inline docs/toc.html]: This change is only cosmetic on the source part, as now the subsections 6.2, 6.3 and 6.4 are aligned at the same column.

**Comment by jlprat** [comment]
CC. @cadonna as you reviewed the last doc changes on formatting and styling, you might want to review these as well. I split the changes in several PRs so it's more bearable to review.

File: docs/toc.html [hunk 2]
```diff
@@ -86,26 +86,26 @@
...
+                        <li><a href="#georeplication-overview">Geo-Replication Overview</a></li>
+                        <li><a href="#georeplication-flows">What Are Replication Flows</a></li>
+                        <li><a href="#georeplication-mirrormaker">Configuring Geo-Replication</a></li>
+                        <li><a href="#georeplication-starting">Starting Geo-Replication</a></li>
+                        <li><a href="#georeplication-stopping">Stopping Geo-Replication</a></li>
+                        <li><a href="#georeplication-apply-config-changes">Applying Configuration Changes</a></li>
+                        <li><a href="#georeplication-monitoring">Monitoring Geo-Replication</a></li>
                     </ul>
-		<li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
+		        <li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
```
Comment (showuon) [inline docs/toc.html]: I think this 6.4 line should align with 6.3 `li` tag, right? (also the following indent)

**Review by showuon — CHANGES_REQUESTED** [review]
@jlprat , thanks for the PR. Left a comment. Thank you.

File: docs/toc.html (hunk 2 above)
Comment (jlprat) [reply docs/toc.html]: You are right, on my IDE it was visually correct, but it was because that line contained 2 tabs, now they are replaced with spaces.

**Review by bbejeck — APPROVED** [review]
Thanks for the improvements @jlprat LGTM.

I validated the changes running a local Apache server

**Comment by jlprat** [comment]
Thanks for the review @bbejeck

**Comment by bbejeck** [comment]
@jlprat could do open an identical PR on https://github.com/apache/kafka-site targeting the /28 directory?  

Thanks!

**Comment by jlprat** [comment]
Sure thing @bbejeck . I'll do it tomorrow morning CEST.

**Comment by jlprat** [comment]
Hi @bbejeck, PR is ready on https://github.com/apache/kafka-site/pull/359

BTW, I did some more doc fixes in the past, and still have a couple of PRs open fixing docs. Would it make sense to also port those to `kafka-site` folder /28?

**Pull request merged**